### PR TITLE
Added my package.

### PR DESCRIPTION
**Pull request by AvexCodes**
Deno Chalk.

### CODE REVIEW

File: src/database.json [hunk 1]
```diff
@@ -881,5 +881,11 @@
     "owner": "chiefbiiko",
     "repo": "xchacha20-poly1305",
     "desc": "XChaCha20-Poly1305 as defined by the XChaCha20 IRTF CFRG draft, and IETF RFC 8439"
+  }, 
+  "deno_chalk": {
+    "type": "github",
+    "owner": "AvexCodes",
+    "repo": "denochalk",
+    "desc": "Style your console with colours and backgrounds like a boss, with Deno Chalk!"
   }
```
Comment (lucacasonato) [inline src/database.json]: The database.json file is alphabetically sorted by package name. Could you insert your package into the correct location in the database.json file?

**Comment by hayd** [comment]
It's confusing that this lib has a completely different API to chalk...

**Comment by crowlKats** [comment]
also it has 100% same functionality as deno's std colors module

**Comment by ry** [comment]
stale

**Pull request closed without merging**